_usage/interoperability.md

```diff
@@ -81,7 +82,7 @@ Interoperability aspects: schema names; identifiers; permissions.
 
 - When the PostgreSQL connection also modifies the Babelfish-created tables or performs DDL to modify such objects, then also transactional semantics aspects will apply.
 
-- A more complex scenario is when a PostgreSQL connection executes a T-SQL procedure (either directly or via a PostgreSQL procedure), or modifies a Babelfish-created table which has a Babelfish-created trigger. The resulting semantics should be carefully validated by the Babelfish user. 
+- A more complex scenario is when a PostgreSQL connection executes a T-SQL procedure (either directly or via a PostgreSQL procedure), or modifies a Babelfish-created table which has a Babelfish-created trigger. The resulting semantics should be carefully validated by the Babelfish user. Beginning with Babelfish v.3.3.0, Babelfish intentionally blocks these configurations to prevent such issues.
```

Comment:
We are only blocking trigger use case, we still allow calling TSQL procedures from PG connection.

Reply:
@robverschoor  Shall we change last line to "Beginning with Babelfish v.3.3.0, Babelfish intentionally blocks modifying a Babelfish-created table which has a Babelfish-created trigger to prevent such issues." ?